### ISSUE TITLE
Questions about continue trainning

### ISSUE THREAD

**Issue by LaLaLailalai** [DegardinBruno/Kinetic-GAN#10]
Hi Bruno,

Do you know how to continue training after it stops in the middle (e.g., 50/200epochs)?

I tried to continue to train the model by myself, but I only found the load weights codes in generate.py.

**Comment by LaLaLailalai** [comment]
By the way, how to set the parameters of evaluation to reproduce the exact FID result?

For example, if I would like to reproduce the NTU120 xsub FID, I should run the command :
"python generate.py --model model_path  --n_classes number_classes  --label class_index  --gen_qtd how_many_samples  "

How to set gen_qtd for each dataset?

**Comment by DegardinBruno** [comment]
Hey @LaLaLailalai, sorry for my late reply!

> Do you know how to continue training after it stops in the middle (e.g., 50/200epochs)?

You can insert the same code with strict=True, where you are ensuring the exact same arch:
```
# Load Models
generator.load_state_dict(torch.load(opt.model), strict=True)
```
And then `for epoch in range(50, opt.n_epochs):` to start at epoch 50. We do not use any scheduler or epoch-dependent parameter, so it must work without a problem!

> By the way, how to set the parameters of evaluation to reproduce the exact FID result?

Just change `n_classes` to 120 and `mlp_dim` to 8 (same as our paper). We also used `gen_qtd` with 1000. The results may vary a bit due to randomness, but not much! Remember to set `trunc_mode` to - in generating for FID and MMD evaluation.

For any further questions, feel free to ask!
